src/main/java/org/broadinstitute/consent/http/mail/freemarker/NewDAAUploadResearcherModel.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.consent.http.mail.freemarker;
+
+public class NewDAAUploadResearcherModel {
```

Comment:
I think this and `NewDAAUploadSOModel`  are a great cases for a Record class.

Reply:
I remember on my previous email PR, Olivia had suggested this and when changing the Model class to a Record class, I ran into FreeMarker errors. Similarly, when I changed the `NewDAAUploadResearcherModel` and `NewDAAUploadSOModel` to Java records, I got the same FreeMarker errors. Back then, I had created [this ticket](https://broadworkbench.atlassian.net/browse/DCJ-441) in the backlog to change all of our FreeMarkerTemplate models to be defined as Records. I have moved the ticket created from the DUOS project to the DCJ project.